api/src/main/kotlin/eu/codeloop/thehub/jobs/model/JobEntity.kt

```diff
@@ -34,6 +36,14 @@ class JobEntity {
     @Column(name = "is_deleted", nullable = false)
     var isDeleted: Boolean = false
 
+// TODO - how get this value?
+//	@Value("\${thehub.models.JobEntity.approved_at}")
+//	private lateinit var approvedAtDefault: String
+
+    @Column(name = "approved_at", nullable = false)
+//	var approvedAt: OffsetDateTime = OffsetDateTime.parse(approvedAtDefault)
+    var approvedAt: OffsetDateTime = OffsetDateTime.parse("2000-01-01T00:00:00.000Z")
```

Comment:
Because it is quite hard to get `approved_at` from `application.yml` just introduce constant that could be reused

Reply:
I created class `DeafultValues`.